### PR TITLE
CopacImporterTest

### PR DESCRIPTION
**Pull request by zellerdev**
Added a few trivial Tests.
Added a Test for Exception.
Completed given Tests with given resources.
6 JUnit Tests without failure or error.

### CODE REVIEW

**Comment by simonharrer** [comment]
Please have a look at https://codecov.io/github/JabRef/jabref/src/main/java/net/sf/jabref/importer/fileformat/CopacImporter.java?ref=1f46af69bd6c026af2392d9a7f045e332ae00ddc which shows a few cases that are missed by your tests.

File: src/test/java/net/sf/jabref/importer/fileformat/CopacImporterTest.java [hunk 1]
```diff
@@ -21,6 +23,32 @@ public void setUp() throws Exception {
...
+    @Test
+    public void testGetFormatName(){
+        CopacImporter importer = new CopacImporter();
+        Assert.assertEquals("Copac",importer.getFormatName());
+    }
+
+    @Test
+    public void testGetCLIId(){
+        CopacImporter importer = new CopacImporter();
+        Assert.assertEquals("cpc",importer.getCLIId());
```
Comment (simonharrer) [inline src/test/java/net/sf/jabref/importer/fileformat/CopacImporterTest.java]: Please format the file according to the Eclipse rules. The effect would be, among other things, that this line is formatted the following way: `Assert.assertEquals("cpc", importer.getCLIId());`

File: src/test/java/net/sf/jabref/importer/fileformat/CopacImporterTest.java [hunk 2]
```diff
@@ -21,6 +23,32 @@ public void setUp() throws Exception {
...
+
+
+    @Test
+    public void testImportEntriesException() throws IOException {
+        thrown.expect(IOException.class);
+
+        CopacImporter importer = new CopacImporter();
+
+        @SuppressWarnings("unused")
+        List<BibtexEntry> entries = importer.importEntries(null, new OutputPrinterToNull());
```
Comment (simonharrer) [inline src/test/java/net/sf/jabref/importer/fileformat/CopacImporterTest.java]: Can you please write this without an assignment? Then you do not need the `@SuppressWarnings` annotation anymore.

File: src/test/java/net/sf/jabref/importer/fileformat/CopacImporterTest.java (hunk 1 above)
Comment (koppor) [inline src/test/java/net/sf/jabref/importer/fileformat/CopacImporterTest.java]: @zellerdev Do you use eclipse and did you create the project using `gradlew eclipse`? Then, the formatting should be automatically be OK, shouldn't it?

**Comment by matthiasgeiger** [comment]
Thanks for contributing! :wink:

Another quick tip: Please ensure that your git config includes a mail adress which is linked with github. This is a prerequisite for your contributions to show up in the github contributors list. See [GitHub Help](https://help.github.com/articles/why-are-my-contributions-not-showing-up-on-my-profile/) and [git manual](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup#Your-Identity).

**Comment by matthiasgeiger** [comment]
May I ask you, why you have closed this pull request (PR)?

**Comment by koppor** [comment]
We had an internal email discussion. The issue of this PR is that it originates from the `master` branch and one cannot change the source branch. Therefore, a new PR will be opened with a more speaking source branch name.

**Comment by koppor** [comment]
Superseeded by #310.

**Pull request closed without merging**